js/script.js

```diff
@@ -0,0 +1,230 @@
+var abaActived = 0;
+var full = [];
+var search = [];
+
+showUsersTable();
+
+searchInput.onkeyup = function(){
+	search = [];
+	if(abaActived == 0){
+		full.map(
+			function(item, index){
+				if(item.id.toString().toLowerCase().indexOf(searchInput.value.toLowerCase()) != -1 || item.name.toLowerCase().indexOf(searchInput.value.toLowerCase()) != -1 || item.email.toLowerCase().indexOf(searchInput.value.toLowerCase()) != -1){
+					search.push(item);
+				}
+			});
+		mountTableUser(search);
+	}else{
+		full.map(
+			function(item, index){
+				if(item.id.toString().toLowerCase().indexOf(searchInput.value.toLowerCase()) != -1 || item.title.toLowerCase().indexOf(searchInput.value.toLowerCase()) != -1){
+					search.push(item);
+				}
+			});
+		mountTablePublication(search);
+	}
+}
+
+abaUsers.onclick = function(){
+	showUsersTable();
+	abaActived = 0;
+	searchInput.value = "";
+}
+
+abaPublications.onclick = function(){
+	showPublicationTable();
+	abaActived = 1;
+	searchInput.value = "";
+}
+
+function showUsersTable(){
+	httpRequest = new XMLHttpRequest();
```

Comment:
Add the "let", "const" or "var" keyword to this declaration of "httpRequest" to make it explicit.